src/views/validated-designs/index.tsx

```diff
@@ -20,26 +59,54 @@ export default function ValidatedDesignsLandingView({
 	categoryGroups,
 }: ValidatedDesignsLandingProps) {
 	return (
-		<BaseLayout mobileMenuSlot={<MobileMenuLevelsGeneric />}>
+		<BaseLayout mobileMenuSlot={<MobileMenuLevelsGeneric />} className={s.root}>
 			<Head>
 				<meta name="robots" content="noindex, nofollow" />
 			</Head>
-			<h1>{'HashiCorp Validated Designs'}</h1>
-			<ul>
+			<LandingHero
+				heading="HashiCorp Validated Design"
+				isHvd={true}
+				className={s.hvdHero}
+			/>
+			<div className={s.categoryGroupsContainer}>
 				{categoryGroups.map((category: HvdCategoryGroup) => (
-					<li key={category.slug}>
-						<h2>{category.title}</h2>
-						<p>{category.description}</p>
-						<ul>
+					<Card
+						elevation="base"
+						key={category.slug}
+						className={s.categoryGroupContainer}
+					>
+						<div className={s.categoryGroupHeader}>
```

Comment:
For a11y reason let's turn this div into a section, `role="region",` that has it's `aria-label` as the `category.title`.

Reply:
Fixed in https://github.com/hashicorp/dev-portal/pull/2279/commits/20487b9a309cfdfbeb19bf2eec2ca7a6f5205cf3

Reply:
@heatlikeheatwave also add the aria-label to it or else it isn't exposed as a landmark when traversing through the accessibility tree with ATs.